engine/dlib/src/dlib/hash.cpp

```diff
@@ -668,3 +668,9 @@ DM_DLLEXPORT const char* dmHashReverseSafe64(uint64_t hash)
     return s != 0 ? s : "<unknown>";
 }
 
+DM_DLLEXPORT const char* dmHashReverseSafe32(uint32_t hash)
+{
+    const char* s = (const char*)dmHashReverse32(hash, 0);
+    return s != 0 ? s : "<unknown>";
+}
+
```

Comment:
Whenever we present our hashed strings, we should use a "Safe" version that always gives us a printable string.